### PR TITLE
Add refmt command to client/package.json

### PR DESCRIPTION
**Pull request by feihong**
I added new refmt command to `client/package.json` because it seemed like existing pre-commit hook wasn't working anymore.

- [x] Add refmt command
- [x] Remove `husky` and `lint-staged` items from package.json
- [x] Reformat some Reason source files 
- [x] Add stop task to server Makefile
- [x] Fix bug in docker-compose.yml where graphql-engine might not start if it can't immediately connect to postgres

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -18,7 +18,8 @@
     "ci:cypress:staging": "CYPRESS_GRAPHQL_ENDPOINT=https://rtop-server.herokuapp.com/v1alpha1/graphql cypress run --record --browser chrome -c baseUrl=\"https://staging.sketch.sh\",defaultCommandTimeout=20000",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:run:record": "cypress run --record --browser chrome"
+    "cy:run:record": "cypress run --record --browser chrome",
+    "refmt": "yarn bsrefmt --in-place $(find src shared src_editor src_analyzer src_embed src_worker -type f \\( -name *.re -o -name *.rei \\))"
```
Comment (jchavarri) [inline client/package.json:22]: @feihong I added some command in https://github.com/Sketch-sh/sketch-sh/pull/333, where I was including opam to fix integration with vscode.

Now there would be two ways to format the code, do you think we should use `bsrefmt` or plain `refmt`?

Comment (feihong) [reply client/package.json:22]: I'm not clear on the difference between bsrefmt and refmt, I thought one was an alias for the other

Comment (jchavarri) [reply client/package.json:22]: Ok, let's use `bsrefmt` (less chances for breakages due to different versions).

Please remove the `format` and `format-check` commands in makefile:

https://github.com/Sketch-sh/sketch-sh/blob/329566784064ca216eec05b98d510568f7db52f4/client/Makefile#L35-L41

Comment (feihong) [reply client/package.json:22]: @jchavarri It seems that [bsrefmt is recommended over refmt](https://reasonml.chat/t/bsrefmt-or-refmt-with-vscode-reason-vscode-1-5-2/1470/4)

> BuckleScript occasionally copies over newer refmt version whenever there were important updates. For consistency reasons, it’s usually recommended to go with the bsrefmt over any other globally installed refmt version (makes it easier to have reproducible results on CI and your coworkers machine).

File: client/package.json [hunk 2]
```diff
@@ -97,17 +98,6 @@
       "/public/"
     ]
   },
-  "husky": {
```
Comment (jchavarri) [inline client/package.json:100]: Shouldn't we remove `husty` and `lint-staged` from dev deps as well?

**Review by jchavarri — COMMENTED** [review]
just a couple of things, the removal of unused dev deps and unused `format*` commands, thanks!

File: client/package.json (hunk 1 above)
Comment (feihong) [reply client/package.json:22]: Removed unused format commands in https://github.com/Sketch-sh/sketch-sh/pull/323/commits/11dda1d2dbc781af8ef3480bdd3b1c452a7e2d69

File: client/package.json (hunk 2 above)
Comment (feihong) [reply client/package.json:100]: Removed unused deps in https://github.com/Sketch-sh/sketch-sh/pull/323/commits/11dda1d2dbc781af8ef3480bdd3b1c452a7e2d69

**Comment by jchavarri** [comment]
Thanks!

**Pull request merged**